media_driver/linux/common/os/mos_utilities_specific.c

```diff
@@ -1932,6 +1933,7 @@ MOS_STATUS MOS_UserFeatureGetValue_AndroidProp(
        eStatus = MOS_STATUS_FILE_NOT_FOUND;
     }
 finish:
+    MOS_SafeFreeMemory(UserFeatureData.StringData.pStringData);
```

Comment:
this memory won't be used after this return?
it will be freed even under success case with your change.